auth/basics/js.md

```diff
@@ -93,15 +93,15 @@ skygear.auth.signupWithUsername(username, password).then((user) => {
   console.log(user); // user object
 }, (error) => {
   console.error(error);
-  if (error.error.code === skygearError.Duplicated) {
+  if (error.error.code === skygear.ErrorCodes.Duplicated) {
```

Comment:
Oh I see, I think we need to specify this for those who are using CDN config, because it would error out if they try to use skygearError.